main.go

```diff
@@ -1,7 +1,16 @@
 package main
 
-import "github.com/opdev/opcap/cmd"
-
+import (
+	"github.com/opdev/opcap/cmd"
+	"go.uber.org/zap"
+	log "github.com/opdev/opcap/internal/logger"
+)
 func main() {
+	mainLogger := log.InitLogger()
```

Comment:
Highly against any of this.... This does not belong in main(). Should be in a PersistentPreRunE in root.go.

Reply:
Removed and initialized all the stuff inside the logger package.